### DISCUSSION TITLE
https://theme-hope.vuejs.press/faq/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#593] [Announcements]
# https://theme-hope.vuejs.press/faq/

Frequency asked questions and corresponding solutions using the theme are included here. Catalog Troubleshooting (troubleshooting.md); Common Errors (common-error.md); Vite Prob...

https://theme-hope.vuejs.press/faq/

**Comment by SunnyGPT** [comment]
Are you there?
